plugins/tekton/src/components/common/Status.tsx

```diff
@@ -100,7 +97,11 @@ export const Status = ({
       return (
         <StatusIconAndText
           {...statusProps}
-          icon={<YellowExclamationTriangleIcon />}
+          icon={
+            <ExclamationTriangleIcon
+              style={{ color: 'var(--pf-v5-global--warning-color--100)' }}
```

Comment:
@debsmita1 can we create separate icon classes and use them as needed in `Status` and `StatusIcon` similar to topology so that in future if we want to change the styles they can be applied to all the places these icons are being used? Also another thing I noticed is that we use `fill` property in topology but you are using `color` here, any reason for this inconsistency or can we use same property to apply icon colors in both plugins?